.vscode/settings.json

```diff
@@ -1,4 +1,36 @@
 {
-	"editor.defaultFormatter": "biomejs.biome",
-	"editor.tabSize": 4
+	"prettier.enable": false,
+	"[html]": {
+		"editor.defaultFormatter": "biomejs.biome"
+	},
```

Comment:
Adds vscode config for formatting with biome

https://github.com/user-attachments/assets/a873e884-3edb-4c8a-8993-bcdbc4fb2757

Reply:
omg i feel seen, someone actually included a nvim plugin

Reply:
Lol, I use neovim too so I figured I'd share what I used